plugins/ci/pkg/ci/ci.go

```diff
@@ -424,7 +424,12 @@ func getConfigurationForClonedRepo() (string, string, *models.Configuration, err
 		return "", "", nil, errors.New("IMAGE_VERSION environment variable not set")
 	}
 
-	basePath := filepath.Join("/app", "repository")
+	var basePath string = os.Getenv("CI_BASE_PATH")
```

Comment:
This helps run the CI plugin outside of its typical docker container, for development / testing.

Reply:
You could possibly move this to `(c *Configuration) SetDefaults()` that is called for both `auto-scan` and on `CI runners` - if that makes sense to do so...

Reply:
That relies on the Insights API, correct?

Reply:
nvm, I've just checked and doesn't really make sense since the default value for `basepath` is different when using `auto-scan` and `CI`

- default `basePath` on auto-scan is `/app/repository`
- default `basePath` for CI is `.`

So we can't really unify the code that sets the `basepath`...